diamond-api/Cargo.toml

```diff
@@ -20,3 +20,4 @@ env_logger = "0.6"
 futures = "0.1"
 whisper = { path = "../whisper" }
 nom = "4"
+bytes = "0.4"
```

Comment:
it should be moved into `[dev-dependencies]` section

Reply:
Done